packages/reports/src/Containers/open-positions.tsx

```diff
@@ -177,12 +142,12 @@ const MobileRowRenderer = ({
     onClickSell,
     measure,
     ...props
-}: TMobileRowRendererProps) => {
+}: TMobileRowRenderer) => {
     React.useEffect(() => {
         if (!is_footer) {
             measure?.();
         }
-    }, [row.contract_info?.underlying, measure, is_footer]);
+    }, [row.contract_info.underlying, measure, is_footer]);
```

Comment:
```suggestion
    }, [row.contract_info?.underlying, measure, is_footer]);
```
nit but important :)